ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_searching_sorting.py

```diff
@@ -468,3 +468,44 @@ def test_jax_numpy_unique(fn_inputs, frontend, test_flags, fn_tree, on_device):
         return_counts=return_counts,
         axis=axis,
     )
+
+
+@handle_frontend_test(
+    fn_tree="jax.numpy.union1d",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        num_arrays=2,
+        shared_dtype=True,
+        min_num_dims=1,
+        max_num_dims=2,
```

Comment:
why limit the input generation so severely? I think it's best if we test for extreme cases as that will help iron out any subtle corner cases that might go unnoticed otherwise. 

Is there a particular reason you've chosen these values for the `max_num_dims` as well as the `min_value` and `max_value` parameters?